keyboards/tr60w/config.h

```diff
@@ -0,0 +1,58 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
```

Comment:
```suggestion
#define PRODUCT_ID      0x4140
```
This is the first 4 numbers from the sha1 hash of the name

Reply:
done.